src/plugins/ssh/init.py

```diff
@@ -49,9 +86,20 @@ def main():
     cmd_params = [jobssh]
 
     if "userssh" in parameters:
-        if "type" in parameters["userssh"] and "value" in parameters["userssh"]:
+        # get user public keys from rest server
+        application_token = plugin_config.get("application_token")
+        username = os.environ.get("PAI_USER_NAME")
+        public_keys = []
+        if application_token:
```

Comment:
Please catch except and return an error code and log something here

Reply:
> Please catch except and return an error code and log something here

Added